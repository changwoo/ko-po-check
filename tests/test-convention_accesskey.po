# -*-text-*-
# test 1 - 접근키 (access key, mnemonic) 검사
#
# * 접근키는 대문자 이어야 한다.
# * 번역문에 괄호 안에 한 글자의 접근키가 들어 있으면 접근키 번역으로 본다.
#
msgid ""
msgstr ""
"Project-Id-Version: test 1\n"
"POT-Creation-Date: 2002-07-24 22:09+0900\n"
"PO-Revision-Date: 2002-07-24 22:09+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# 대문자 그놈 접근키
#: test.c:1
msgid "_File"
msgstr "파일(_F)"

# E: 소문자로 잘못 번역한 그놈 접근키
#: test.c:1
msgid "T_ool"
msgstr "툴(_o)"

# 올바른 그놈 접근키
#: test.c:1
msgid "Save _as..."
msgstr "다른 이름으로 저장(_A)..."

# 대문자 KDE 접근키
#: test.c:1
msgid "&Edit"
msgstr "편집(&E)"

# E: 소문자로 잘못 번역한 KDE 접근키
#: test.c:1
msgid "C&ut"
msgstr "잘라내기(&u)"

# 올바른 KDE 접근키
#: test.c:1
msgid "Pre&vious"
msgstr "이전(&V)"

# 다른 접근키
#: test.c:1
msgid "S_earch"
msgstr "찾기(_S)"

# 접근키가 아님
#: test.c:1
msgid "_filename"
msgstr "(_파일 이름)"

# 접근키가 아님 2
#: test.c:1
msgid "(test_N)"
msgstr "(테스트_N)"

# 접근키가 아님 3
#: test.c:1
msgid "pre-ANSI style (K&R)"
msgstr "ANSI 이전 스타일 (K&R)"

# 접근키가 아님 (pango label)
#: test.c:1
msgid "_Chicken &amp; egg"
msgstr "닭이냐 달걀이냐(_D)"

# 접근키가 아님 (kde label)
#: test.c:1
msgid "(test_N)"
msgstr "(테스트_N)"

# 접근키가 아님 (GNOME reserved)
#: test.c:1
msgid "translator_credits"
msgstr "홍길동"

# 접근키가 아님 (KDE URL)
#: test.c:1
msgid "See http://xxx.xxx/xxx?page=1&lang=en"
msgstr "보세요  http://xxx.xxx/xxx?page=1"
