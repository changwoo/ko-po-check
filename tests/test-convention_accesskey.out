-:26: "(_o)": 접근키가 소문자입니다
-:41: "(&u)": 접근키가 소문자입니다
-:51: 'E' vs 'S': 원문과 번역문의 접근키가 다릅니다
-:56: 번역문에 접근키가 없거나 두 개 이상입니다
-:71: 'C' vs 'D': 원문과 번역문의 접근키가 다릅니다
